lib/foreman_puppet/engine.rb

```diff
@@ -80,6 +80,8 @@ def argument(name, type, *rest, loads: nil, **kwargs, &block)
       ::TemplateCombination.include ForemanPuppet::Extensions::TemplateCombination
       ::ProvisioningTemplate.include ForemanPuppet::Extensions::ProvisioningTemplate
 
+      ::HostCounter.prepend ForemanPuppet::Extensions::HostCounter
```

Comment:
Should we do `include` here

and have a module Extensions that is prepended in the concern?

Reply:
I don't see the value in that... if the only method we are patching needs to be prepended :thinking:

Reply:
It would just be consistency. But I'm fine if we leave it like this for now.